packages/components/Atoms/Global/scss/tools/_typography.scss

```diff
@@ -9,17 +9,32 @@
     font-size: $font-size;
   }
   line-height: $line-height;
-  // The custom Vicfont has fixed font weight,
-  // so we set this to always normal,
-  // this will make all browsers render the font more consistently.
-  // See: https://stormconsultancy.co.uk/blog/development/tips-tricks/dealing-with-firefox-rendering-fonts-too-bold-with-browser-specific-css/
-  font-weight: normal;
+  @if $include-base == true {
+    // The custom Vicfont has fixed font weight,
+    // so we set this to always normal,
+    // this will make all browsers render the font more consistently.
+    // See: https://stormconsultancy.co.uk/blog/development/tips-tricks/dealing-with-firefox-rendering-fonts-too-bold-with-browser-specific-css/
+    font-weight: normal;
+  }
 }
 
-@mixin rpl_display_background($font-size) {
-  background-color: rpl-color('secondary');
-  color: rpl-color('white');
-  display: inline;
+@mixin rpl_display_background($font-size, $include-base: true) {
+  @if $include-base == true {
+    background-color: rpl-color('secondary');
+    color: rpl-color('white');
```

Comment:
Let's use this PR to move hardcoded color out of this mixin, it's hard to customised. I will suggest to use sass variables like `$rpl-display-background-color` and `$rpl-display-text-color`.
We should never use `rpl-color` directly, always use variable.

Reply:
Will fix.